setup.cfg

```diff
@@ -1,8 +0,0 @@
-[flake8]
-ignore=E501
-
-[tool:pytest]
-addopts=-n=auto --verbose -r=a --junit-xml=results/junit.xml --html=results/index.html
-xfail_strict=true
-base_url=https://addons-dev.allizom.org
-sensitive_url=mozilla\.org
```

Comment:
I've moved this into `tox.ini` so to reduce the number of places we have such config.